ci-automation/ci-config.env

```
@@ -53,3 +53,27 @@ QEMU_BIOS="/usr/share/qemu/bios-256k.bin"
 # Published by vms.sh as part of the qemu vendor build.
 QEMU_UEFI_BIOS="flatcar_production_qemu_uefi_efi_code.fd"
 
+
+# -- Equinix Metal --
+EQUINIXMETAL_PARALLEL="${PARALLEL_TESTS:-4}"
+# Metro is a set of Equinix Metal regions
+EQUINIXMETAL_amd64_METRO="SV"
+EQUINIXMETAL_arm64_METRO="DA"
+# Name of the Equinix Metal image
+EQUINIXMETAL_IMAGE_NAME="flatcar_production_packet_image.bin.bz2"
+# Storage URL required to store user-data
+EQUINIXMETAL_STORAGE_URL="gs://flatcar-jenkins/mantle/packet"
+# Equinix Metal default AMD64 instance type
+EQUINIXMETAL_amd64_INSTANCE_TYPE="c3.small.x86"
```

```suggestion
EQUINIXMETAL_amd64_INSTANCE_TYPE="${EQUINIXMETAL_amd64_INSTANCE_TYPE-c3.small.x86}"
```
I think it would be good to be able to overwrite it like we can do now in the original pipeline